packages/@coorpacademy-components/src/molecule/video-player/jwplayer.js

```diff
@@ -142,6 +150,8 @@ class JWPlayer extends React.Component {
 }
 
 JWPlayer.propTypes = {
+  autostart: PropTypes.bool,
```

Comment:
I don't understand why there is too prop types `autostart`, can you explain please ?

Reply:
naming issue you're right this one should be
```
 disableAutostart: PropTypes.bool,
```